src/mlagility/api/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Getting image from docker hub
```

Comment:
Interestingly, the Docker implementation is faster than the local one. Maybe a newer version of `onnxruntime` is being used? (nonblocking)

For `bert.py` in `ms`:

| Local | container | 
|------|-----------|
| 344.7 | 313.5 |
| 343.5 | 440.0 |
| 345.3 | 309.6 |
| 342.7 | 311.3 |

I also noticed that the benchmark results were 440ms for one of the runs using the container (40% higher than other values). Not sure if this is related to the container or something that could happen without it.

If we thing this is not related to the container, I would recommend opening an issue for measuring and checking the standard deviation between benchmarks of the same model.

Reply:
Sounds good. I agree that this should not be flagged as an issue now.